### DISCUSSION TITLE
A needed fix for "chromote"

### DISCUSSION THREAD

**Discussion by cellocgw** [r-universe-org/help#485] [Ideas]
I just read the blog entry [https://ropensci.org/blog/2024/09/10/script-screenshots/](url)   and noticed the infamous "anti-pattern"  use of "sleep()" to hope&pray a call is complete.  Isn't there a way to query some status item the Chrome instance to see that page loading is finished?

**Comment by jeroen** [comment]
This is not really a question on r-universe but rather a question for blog author @maelle or the authors of the chromote package: https://github.com/rstudio/chromote/issues

**Reply by cellocgw** [reply]
OK, thanks -- navigating  github's forum setup is a mess.
